index.html

```diff
@@ -4100,10 +4105,12 @@ <h2><span class="chunk">cICP</span>
   of computer graphics and web images, including those used in traditional PNG
   workflows, are <a>full-range images</a>.  If <code>Video Full Range
   Flag</code> value is <code>0</code>, then the image is a <a>narrow-range
-  image</a>. Such images are found in video workflows and the interpretation of
-  sample values smaller (larger) than reference black (white) depend on the
-  application. [[SMPTE RP 2077]] specifies mapping between <a>full-range
-  images</a> and <a>narrow-range images</a>.</aside>
+  image</a>. Narrow range images are found in video workflows where the interpretation of
+  sample values below 0% signal level or above 100% signal level exist to avoid clipping of
```

Comment:
I do not think 0% and 100% of signal level is sufficiently descriptive, i.e. 100% of what?

Reply:
I've added additional text to describe narrow range

- Undershoot/Overshoot
- signal range (instead of level) so that 0-100% has better context

Reply:
Could we give an example?  

Narrow range images are found in video workflows where the interpretation of sample values below Black (0%) signal level or above Peak White 100% signal level - E.g. in ITU-R BT.709 10 bit video, 0% is defined as code value 64 and 100% as code value 940.

Such over- and under-shoots exist to allow temporary excursions in uncontrolled lighting conditions, and artifacts caused by video filtering

to be stored without clipping which can improve image quality during additional stages of compression. Narrow Range can also preserve additional color volume (both light and color)  as described in [[EBU R 103]]. [[SMPTE RP 2077]] specifies mapping between <a>full-range  images</a> and <a>narrow-range images</a> and protected code values for SDI(baseband) carraige.